android-wsbridge/app/src/main/java/com/dji/wsbridge/lib/BridgeApplication.java

```diff
@@ -51,6 +44,23 @@ public Bus getBus() {
         return this.eventBus;
     }
 
+    public static boolean isInternalVersion() {
```

Comment:
These would fit better in a separate utility class.

Reply:
Moved